lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -1775,7 +1775,7 @@ def test_certificate_generation_for_students(self):
             'failed': 3,
             'skipped': 2
         }
-        with self.assertNumQueries(168):
+        with self.assertNumQueries(184):
```

Comment:
Note to reviewers: there are 8 different learners (increase of 2 queries per learner).